### PR TITLE
Add contributing-guides/alias-pages.md

### PR DESCRIPTION
**Pull request by marchersimon**
Ref: #5986 
I hope the page structure is fine.

### CODE REVIEW

File: contributing-guides/alias-pages.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
+# Alias pages
+
+To indicate that a command links to another command, we use alias pages.
+This file contains a list of all translations of an alias page template.
+The templates can be chnaged in cases where it makes sense.
```
Comment (bl-ue) [inline contributing-guides/alias-pages.md:5]: `In order to document a command which is an alias of another command, you can write an alias page.` as an introductory sentence is more appealing to me—thoughts? 
```suggestion
In order to document a command which is an alias of another command, you can
write an alias page. This file contains a list of all translations of the alias
page template decided upon in
[#5368](https://github.com/tldr-pages/tldr/pull/5368).
The templates can be changed when necessary.
```

**Comment by marchersimon** [comment]
I added a mention to the `alias-pages.md` file in `CONTRIBUTING.md` in bf6bcc3.

File: contributing-guides/alias-pages.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+Not translated yet.
+
+### fa
+
+Not translated yet.
+
+### fr
+
+Not translated yet.
```
Comment (patricedenis) [inline contributing-guides/alias-pages.md:78]:
```suggestion
# exemple

> Cette commande est un alias de `exemple`.

- Voir la documentation de la commande originale :

`tldr exemple`
```
```

**Review by patricedenis — COMMENTED** [review]
I have translated the French section

**Comment by bl-ue** [comment]
Great @navarroaxel & @patricedenis!  Will you please commit those directly to the PR's branch instead of via suggestions, however?  Because if you look at the English example, you'll see that it's wrapped in a code block, styled as `markdown`.  Since GitHub suggestions use the same syntax (three backticks \`\`\`), you can't add make your suggestions add code, so when they get applied they'd actually not be in code in the file itself.  You can edit the page right [here](https://github.com/marchersimon/tldr/edit/alias-translations/contributing-guides/alias-pages.md).

**Comment by bl-ue** [comment]
Pinging users who have contributed to translations
- [ ] Bosnian (`bs`) - @Vukory
- [ ] Danish (`da`) - @MrBurrBurr
- [ ] Persian (`fa`) - @opoet7
- [ ] Hindi (`hi`) - @2KAbhishek, @Athi223
- [ ] Indonesian (`id`) - @irhamdz, @widyakumara, @levirs565, @mul14, @reinhart1010
- [ ] Italian (`it`) - @mebeim, @glenacota
- [ ] Japanese (`ja`) - @yu-ichiro, @nlsum1, @ktrueda, @Miyuutsu, @shu-pf
- [ ] Korean (`ko`) - @Proscream, @JJByun, @lch7167, @dmakth
- [ ] Malayalam (`ml`) - @abrahamparayil, @govind-shenoy
- [ ] Dutch (`nl`) - @Waples
- [ ] Norwegian (`no`) - @JoeStone13, @sindresorhus?
- [ ] Polish (`pl`) - @mbiesiad, @HakierGrzonzo, @lemieszek, @mrskizzex, @chanibal
- [ ] Russian (`ru`) - @Bodyhealer, @NikonP, @subzey, @Ryzhehvost
- [ ] `hbs` - @zlatanvasovic
- [ ] Swedish (`sv`) -  @Yrsa97
- [ ] Tamil (`ta`) - @arunisaac, @MohamedSabthar, @seenukarthi
- [ ] Thai (`th`) - @nsuthison
- [ ] Turkish (`tr`) - @yutyo
- [ ] Chinese (`zh`) - @gyli @noarchwastaken @zhouLion @starccy

**Comment by 2KAbhishek** [comment]
Hi @bl-ue, received your ping 😁
Are any changes necessary in translations?

File: contributing-guides/alias-pages.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+```Markdown
+# example
+
+> This command is an alias of `example`.
+
+- View documentation for the original command:
+
+`tldr example`
+```
```
Comment (bl-ue) [inline contributing-guides/alias-pages.md:46]: Hi @2KAbhishek! Yes, can we have a freshly crafted Hindi translation for this snippet (between the \`\`\`)?

Comment (2KAbhishek) [reply contributing-guides/alias-pages.md:46]: I'm on it!

Comment (bl-ue) [reply contributing-guides/alias-pages.md:46]: Just send a message right here...no need to open a PR or anything like that ;)

Comment (2KAbhishek) [reply contributing-guides/alias-pages.md:46]:
```
# उदाहरण

> यह आदेश `उदाहरण` का उपनाम है।

- मूल आदेश के लिए दस्तावेज़ देखें:

`tldr उदाहरण`
```

Comment (2KAbhishek) [reply contributing-guides/alias-pages.md:46]: here you go @bl-ue

**Comment by CleanMachine1** [comment]
Why do we not have an arabic version, seems logical. How many of our users speak it?

**Comment by Waples** [comment]
Received the ping, though currently in hospital after a dog attack, will do this this when I'm back at my laptop

**Comment by bl-ue** [comment]
👀 Sounds dreadful! 😬

**Comment by levirs565** [comment]
Received your ping. How to do this?

**Comment by bl-ue** [comment]
I'll tell you.

**Comment by bl-ue** [comment]
Everyone who received the ping, please translate the following snippet into your language:
```markdown
# example

> This command is an alias of `example`.

- View documentation for the original command:

`tldr example`
```
...and send the translation in a new message, like the following (please put three backticks (\`\`\`) at the start and end of the message):

**Comment by bl-ue** [comment]
English translation
```Markdown
# example

> This command is an alias of `example`.

- View documentation for the original command:

`tldr example`
```

**Comment by levirs565** [comment]
Indonesian translation.

```
# contoh

> Perintah ini merupakan alias dari `contoh`.

- Menampilkan dokumentasi untuk perintah asli. 

`tldr contoh`
```

File: contributing-guides/alias-pages.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+
+Not translated yet.
+
+### it
+
+Not translated yet.
+
+### ja
+
+Not translated yet.
```
Comment (ktrueda) [inline contributing-guides/alias-pages.md:118]:
```suggestion
```Markdown
# サンプル

> このコマンドは `サンプル` のエイリアスです.

- オリジナルのコマンドのドキュメントを表示する:

`tldr サンプル`
```
```

**Comment by widyakumara** [comment]
> Indonesian translation.
> 
> ```
> # contoh
> 
> > Perintah ini merupakan alias dari `contoh`.
> 
> - Menampilkan dokumentasi untuk perintah asli. 
> 
> `tldr contoh`
> ```

Looks good except the description should end with a colon instead of a period

**Comment by arunisaac** [comment]
Should the word "example" be translated? Isn't that a command name (a proper noun) that is always in English?

Here's the Tamil translation with example untranslated.

```markdown
# example

> இக்கட்டளை `example` கட்டளையின் மற்றொருப் பெயர்.

- அக்கட்டளையின் விளக்கத்தைக் காண:

`tldr example`
```

**Comment by CleanMachine1** [comment]
@arunisaac You're right there, everyone else's suggestions will edited to change certain parts to command line text (english)

**Comment by bl-ue** [comment]
Yes, let's not translate example now that you point it out, @arunisaac 🤔

**Comment by levirs565** [comment]
Improved Indonesian translation.

```
# example

> Perintah ini merupakan alias dari `example`.

- Menampilkan dokumentasi untuk perintah asli. 

`tldr example`

```

**Comment by nsuthison** [comment]
Here is the Thai translation :)

```
# example

> คำสั่งนี้เป็น alias ของ `example`.

- เรียกดูรายละเอียดสำหรับคำสั่งตัวเต็ม:

`tldr example`
```

**Comment by CleanMachine1** [comment]
Thanks for your Thai contribution!

File: contributing-guides/alias-pages.md (hunk 4 above)
Comment (bl-ue) [reply contributing-guides/alias-pages.md]: → f41ac298cdf10d2c3b62b37183b79648f8c412e8

**Comment by bl-ue** [comment]
Note to maintainers: if you don't mind, please allow me to merge this PR, as I'd like perform a certain action when merging... 😉

**Comment by noarchwastaken** [comment]
Pong. Here's Simplified Chinese:

```markdown
# 示例

> 这是`示例`命令的一个别名。

- 原命令的文档在：

`tldr example`
```

**Comment by noarchwastaken** [comment]
I can do Traditional Chinese too, it's usually just an `opencc` away. But please point out if it's wrong:
```markdown
# 示例

> 這是`示例`命令的一個別名。

- 原命令的文檔在：

`tldr example`
```

**Comment by bl-ue** [comment]
I'll link a few PRs that I found:
- https://github.com/marchersimon/tldr/pull/2
- https://github.com/marchersimon/tldr/pull/3
- https://github.com/marchersimon/tldr/pull/4

@marchersimon will you please merge https://github.com/marchersimon/tldr/pull/2 and https://github.com/marchersimon/tldr/pull/3, but not https://github.com/marchersimon/tldr/pull/4 (because that was supplied [here](https://github.com/tldr-pages/tldr/pull/5988#discussion_r635440953))?

**Comment by usadson** [comment]
Here's the Dutch (nl) translation :)

```
# example

> Dit commando is een alias van `example`.

- Bekijk de documentatie van het originele commando:

`tldr example`
```

**Comment by bl-ue** [comment]
@noarchwastaken we're not supposed to translate example, actually. Can I just change `示例` in the description to `example` or must there be spaces around it since it's English, or not since it's styling is different (`code`)?

**Comment by marchersimon** [comment]
> @marchersimon will you please merge marchersimon#2 and marchersimon#3, but not marchersimon#4 (because that was supplied here)?

Never thought I'd recieve any PRs 😄. I'll merge them right away.

**Comment by noarchwastaken** [comment]
> @noarchwastaken we're not supposed to translate example, actually. Can I just change `示例` in the description to `example` or must there be spaces around it since it's English, or not since it's styling is different (`code`)?

ah yes

**Comment by noarchwastaken** [comment]
Please add a space around.

**Comment by marchersimon** [comment]
I just saw that `example` is also translated in the Polish version, however I already commtited it. @HakierGrzonzo could you please untranslate it?

**Comment by bl-ue** [comment]
@marchersimon it's easy enough to untranslate ourselves: 8c4904783a692ee6c1369be8407d0dd1b53a6fa9

You can do that in the other https://github.com/marchersimon/tldr/pull/3 yourself.

**Comment by CleanMachine1** [comment]
Dont ping the people who have helped, just to ask them to remove 2 words...

**Comment by marchersimon** [comment]
@patricedenis we decided on leaving `example` untranslated, since it's just a placeholder for a command.

**Review by HakierGrzonzo — APPROVED** [review]
Untranslated correctly

File: contributing-guides/alias-pages.md [hunk 5]
```diff
@@ -0,0 +1,284 @@
...
+---
+### fa
+
+Not translated yet.
+
+---
+### fr
+
+```markdown
+# exemple
```
Comment (patricedenis) [inline contributing-guides/alias-pages.md:102]:
```suggestion
# example
```

File: contributing-guides/alias-pages.md [hunk 6]
```diff
@@ -0,0 +1,284 @@
...
+
+> Cette commande est un alias de `exemple`.
+
+- Voir la documentation de la commande originale :
+
+`tldr exemple`
```
Comment (patricedenis) [inline contributing-guides/alias-pages.md:108]:
```suggestion
`tldr example`
```

File: contributing-guides/alias-pages.md [hunk 7]
```diff
@@ -0,0 +1,284 @@
...
+
+Not translated yet.
+
+---
+### fr
+
+```markdown
+# exemple
+
+> Cette commande est un alias de `exemple`.
```
Comment (patricedenis) [inline contributing-guides/alias-pages.md:104]:
```suggestion
> Cette commande est un alias de `example`.
```

**Comment by patricedenis** [comment]
> @patricedenis we decided on leaving `example` untranslated, since it's just a placeholder for a command.

OK, sorry, I did not read that one.
I reverted back to `example`

**Comment by marchersimon** [comment]
Sorry, I messed something up there.

**Comment by CleanMachine1** [comment]
Was wondering why you did that, didn't you mean to close as well?

**Comment by marchersimon** [comment]
I tried what happened if you run `git delete-branch` on the currently checked out branch, since I assumed it would throw an error. I also didn't assume that it would get synced to Github automatically.

**Comment by CleanMachine1** [comment]
Well it does give an error that local couldn't be removed. Remote can still be done

**Comment by bl-ue** [comment]
I would've tested that functionality on a dummy branch if I were you 😬😬😬

**Comment by marchersimon** [comment]
I trusted too much in that it wouldn't work and just wanted to know what error message I would get. Also I was too lazy checking out to another branch 😄

**Comment by CleanMachine1** [comment]
Do you 2 not have a test repo for testing random things?

**Comment by bl-ue** [comment]
Yes, I have https://github.com/bl-ue/github-testing, but it's private like my other ones 😶

File: contributing-guides/alias-pages.md [hunk 8]
```diff
@@ -0,0 +1,284 @@
...
+`tldr example`
+```
+
+---
+### th
+
+```markdown
+# example
+
+> คำสั่งนี้เป็น alias ของ `example`.
```
Comment (marchersimon) [inline contributing-guides/alias-pages.md:248]: Hey @nsuthison, is there a reason you didn't translate `alias`? I found "นามแฝง" as a possible translation.

Comment (nsuthison) [reply contributing-guides/alias-pages.md:248]: @marchersimon `นามแฝง` is not quite the correct translation for this case. I try to translate the word `alias` to many Thai words but all seem hard to understand or feel weird compared to use `alias` directly.

Comment (nsuthison) [reply contributing-guides/alias-pages.md:248]: To give you more context about this word, `นามแฝง` means the name which is not your real name that you'll use when you're undercover or write a book but didn't want to use your real name.

Marchersimon also is a `นามแฝง` as well.

**Comment by arunisaac** [comment]
> To give you more context about this word, `นามแฝง` means the name
> which is not your real name that you'll use when you're undercover or
> write a book but didn't want to use your real name.

The same is true in Tamil as well. There is no exact translation of
'alias' as used in our context of commands. But, here's how I managed:
Instead of translating "This command is an alias for example", I
translated "This command is another name for example". Maybe, you can do
something like that.

Cheers!

File: contributing-guides/alias-pages.md (hunk 8 above)
Comment (nsuthison) [reply contributing-guides/alias-pages.md:248]: As per @arunisaac suggestion, I'll translate it to this

```
# example

> คำสั่งนี้เป็นอีกชื่อหนึ่งของคำสั่ง `example`.

- เรียกดูรายละเอียดสำหรับคำสั่งตัวเต็ม:

`tldr example`
```

@marchersimon

**Comment by subzey** [comment]
Russian:

```md
# example

> Эта команда — псевдоним для `example`.

- Смотри документацию для оригинальной команды:

`tldr example`
```

File: contributing-guides/alias-pages.md (hunk 8 above)
Comment (marchersimon) [reply contributing-guides/alias-pages.md:256]:
```suggestion
> คำสั่งนี้เป็นอีกชื่อหนึ่งของคำสั่ง `example`.
```

**Review by patricedenis — APPROVED** [review]

**Comment by marchersimon** [comment]
Should we merge this for now and open PRs for future translations?

**Comment by CleanMachine1** [comment]
No, why. Give it another week then maybe.

**Comment by bl-ue** [comment]
Re-pinging those whom we need translations from:

- [ ] Danish (`da`) - @MrBurrBurr
- [ ] Persian (`fa`) - @opoet7
- [ ] Italian (`it`) - @mebeim, @glenacota
- [ ] Korean (`ko`) - @Proscream, @JJByun, @lch7167, @dmakth
- [ ] Norwegian (`no`) - @JoeStone13, @sindresorhus?
- [ ] `hbs` - @zlatanvasovic
- [ ] Swedish (`sv`) -  @Yrsa97
- [ ] Turkish (`tr`) - @yutyo

See my [message](https://github.com/tldr-pages/tldr/pull/5988#issuecomment-844611405) above for what to do.

**Comment by bl-ue** [comment]
Hey @vladimyr 👋🏻 are you able to help out here with translations by any chance?

**Comment by comradekingu** [comment]
```
# eksempel

> Denne kommandoen er et alias for `eksempel`.

- Vis dokumentasjonen for den opprinnelige kommandoen:

`tldr eksempel`
```
CLAs are dumb, and it should probably be "nb_NO" and no "no", but in the very least "nb", since there are multiple Norwegian languages. The biggest of which are nn_NO and nb_NO.

**Comment by sbrl** [comment]
@comradekingu: The CLA is unfortunately a necessary thing we need here.

Oh, that's interesting about Norwegian languages. I suggest opening a separate issue about that.

**Comment by marchersimon** [comment]
I'll add it to the `no` section for now, but feel free to open an issue about that if you want to discuss this further @comradekingu

File: contributing-guides/alias-pages.md [hunk 9]
```diff
@@ -0,0 +1,300 @@
...
+- Bekijk de documentatie van het originele commando:
+
+`tldr example`
+```
+
+---
+### no
+
+```markdown
+# eksempel
```
Comment (navarroaxel) [inline contributing-guides/alias-pages.md:190]:
```suggestion
# example
```

File: contributing-guides/alias-pages.md [hunk 10]
```diff
@@ -0,0 +1,300 @@
...
+
+> Denne kommandoen er et alias for `eksempel`.
```
Comment (navarroaxel) [inline contributing-guides/alias-pages.md:192]:
```suggestion
> Denne kommandoen er et alias for `example`.
```

File: contributing-guides/alias-pages.md [hunk 11]
```diff
@@ -0,0 +1,300 @@
...
+
+- Vis dokumentasjonen for den opprinnelige kommandoen:
+
+`tldr eksempel`
```
Comment (navarroaxel) [inline contributing-guides/alias-pages.md:196]:
```suggestion
`tldr example`
```

File: contributing-guides/alias-pages.md (hunk 9 above)
Comment (marchersimon) [reply contributing-guides/alias-pages.md:190]: Totally forgot about that, thanks for noticing

**Comment by mebeim** [comment]
Here you go:

```
# esempio

> Questo comando è un alias per `esempio`.

- Consulta la documentazione del comando originale:

`tldr esempio`
```

**Comment by zlatanvasovic** [comment]
@bl-ue `hbs` is same as `bs`, already done by @Vukory. :)

**Review by zlatanvasovic — APPROVED** [review]
:100:

**Comment by marchersimon** [comment]
Should we merge this? I think if we wait until we have all translations this is going to take very long.

**Review by bl-ue — APPROVED** [review]

**Comment by zlatanvasovic** [comment]
I vote for merge right now. People can make a new PR to contribute to the file any time.

**Review by sbrl — APPROVED** [review]
Let's do this!

**Comment by SethFalco** [comment]
One problem with keeping all templates in a single file, is that it'll be a pain whenever the template changes.
When or if a translation platform will be configured, it'll be easy to handle for normal files because they're monolingual.

When this file changes, tracking the progress will not be possible on the translation platform.

Would it be feasible to instead make a `templates/{{lang}}-alias-page.md` or something so it would be easier to manage when it needs changing? Then when the English one changes, Weblate or whatever else will just detect it and let translators know the other ones need to be changed too.

If we commit it like this, we'll have to make a PR and checklist every time the template is updated.

**Review by nicokosi — APPROVED** [review]
Aliases FTW! :+1:

**Comment by marchersimon** [comment]
@SethFalco I don't think this template will change very often and I don't really like the idea of having another folder in the repository.

**Review by SethFalco — APPROVED** [review]

**Comment by bl-ue** [comment]
Thank you everyone! ❤️

**Pull request merged**